en/docs/references/concepts/authorization/user-managed-access.md

```diff
@@ -74,12 +78,15 @@ The UMA grant is an extension of the OAuth 2.0 grant and enhances OAuth capabili
 
 - **pre-defined authorization grant rules**: the resource owner can configure authorization grant rules (policy conditions) at will rather than having to authorize access token issuance each time synchronously. The authorization server and resource server interact with the client and requesting party asynchronously with respect to the resource owner's interactions. 
```

Comment:
```suggestion
- **predefined authorization grant rules**: the resource owner can configure authorization grant rules (policy conditions) at will rather than having to authorize access token issuance each time synchronously. The authorization server and resource server, interact with the client and requesting party asynchronously, with respect to the resource owner's interactions. 
```